Source/WebInspectorUI/UserInterface/Controllers/DebuggerManager.js

```diff
@@ -82,6 +82,7 @@ WI.DebuggerManager = class DebuggerManager extends WI.Object
         this._breakpointContentIdentifierMap = new Multimap;
         this._breakpointScriptIdentifierMap = new Multimap;
         this._breakpointIdMap = new Map;
+        this._breakpointSourceCodeContentIdentifierMap = new Map;
```

Comment:
NIT: while this is used for breakpoints, it technically doesn't contain any breakpoints, so id just rename it to `_sourceCodeContentIdentifierMap`